### ISSUE TITLE
Potrzebuje pomocy ze sprejami.

### ISSUE THREAD

**Issue by VVrongVVay** [lpiob/MTA-XyzzyRP#373]
Jak malować sprejem na ścianie?

**Comment by XJMLN** [comment]
Nadaj sobie do EQ item 43 (szkicownik), otwórz go narysuj coś w nim i potem jeśli Twoja umiejętność sprayowania (ab_spray) jest większa niż 50 możesz sprayem (UID 25) malować.

**Comment by VVrongVVay** [comment]
A jak zwiększyć umiejetność?

W dniu piątek, 4 listopada 2016 XJMLN <notifications@github.com> napisał(a):

**Comment by VVrongVVay** [comment]
Dobrze, zwiększyłem. Sprawdzę działanie.

W dniu piątek, 4 listopada 2016 Michał Ulejczyk <vvrongvvay@gmail.com>
napisał(a):

**Comment by VVrongVVay** [comment]
Gdy dodaje w bazie danych 50 i robie reconnect to spowrotem mam 0...


W dniu 4 listopada 2016 22:14 użytkownik Michał Ulejczyk <
vvrongvvay@gmail.com> napisał:

**Comment by XJMLN** [comment]
Twoja postać musi być offline.

**Comment by VVrongVVay** [comment]
Podczas nadawania?

2016-11-04 22:40 GMT+01:00 XJMLN <notifications@github.com>:

**Comment by XJMLN** [comment]
Tak, nie możesz być zalogowany na konto które edytujesz.

**Comment by VVrongVVay** [comment]
Mam jeszcze jeden problem.. Jak mam zablokować by do bankomatu nie dało się
wpłacić kwoty 0,5$ tylko najmniej 1$? To samo podczas wymiany
przedmiotami.. Da się przesłać 0,5 przedmiotu.

2016-11-04 22:42 GMT+01:00 XJMLN <notifications@github.com>:

**Comment by XJMLN** [comment]
Musisz dodać sprawdzanie czy kwota jest większa lub równa 1.

**Comment by VVrongVVay** [comment]
Mógłby Pan mi napisać tą linijkę?

2016-11-05 0:49 GMT+01:00 XJMLN <notifications@github.com>:

**Comment by XJMLN** [comment]
> `if string.find(tonumber(kwota),"%p%d") ~= nil or string.find(tonumber(kwota),"%p%d%d") ~= nil then outputChatBox("Nie możesz wpłacić/wypłacić kwoty z groszami.",255,0,0) return end`

[lss-bankomaty](https://github.com/lpiob/MTA-XyzzyRP/blob/master/resources/%5BXyzzyRP%5D/lss-bankomaty/bankomaty_c.lua#L210) tutaj wklejasz, następnie [tutaj](https://github.com/lpiob/MTA-XyzzyRP/blob/master/resources/%5BXyzzyRP%5D/lss-bankomaty/bankomaty_c.lua#L219) oraz pod [tą ](https://github.com/lpiob/MTA-XyzzyRP/blob/master/resources/%5BXyzzyRP%5D/lss-bankomaty/bankomaty_c.lua#L236) linijką.

[[lss-trade](https://github.com/lpiob/MTA-XyzzyRP/blob/master/resources/%5BXyzzyRP%5D/lss-trade/trade.lua#L114)]nad tym wklejasz

>  `if string.find(tonumber(kwota),"%p%d") ~= nil or string.find(tonumber(kwota),"%p%d%d") ~= nil then 
> 	outputChatBox("Nie możesz przelać drugiemu graczu kwoty z groszami, transakcja przerwana.",p1,255,0,0) 
> 	outputChatBox("Transakcja przerwana, drugi gracz próbował przesłać kwotę z groszami",p2,255,0,0)
> 	return 
> end`

 wymiana będzie anulowana jeśli gracz wyśle drugiemu kwotę z groszami. Wzorując się na tym skrawku który podałem poniżej możesz zablokować wszędzie przesył groszy.

**Comment by VVrongVVay** [comment]
Nie działa..


W dniu 5 listopada 2016 01:11 użytkownik XJMLN <notifications@github.com>
napisał:

**Comment by XJMLN** [comment]
W lss-trade zmień tonumber(kwota) na tonumber(v.count).

**Comment by VVrongVVay** [comment]
Ale w bankomatach nie działa.

2016-11-05 1:18 GMT+01:00 XJMLN <notifications@github.com>:

**Comment by VVrongVVay** [comment]
I w lss-trade nie wiem nad czym to wkleić bo nie dał Pan odnośnika do
linijki

2016-11-05 1:19 GMT+01:00 Michał Ulejczyk <vvrongvvay@gmail.com>:

**Comment by XJMLN** [comment]
Nad 114 linijką, co do błędu już sprawdzam.

**Comment by VVrongVVay** [comment]
Niestety ani w trade, ani w bankach nie działa i nadal da się przesyłać 0.5

2016-11-05 1:24 GMT+01:00 XJMLN <notifications@github.com>:

**Comment by XJMLN** [comment]
Skrypt u mnie działa prawidłowo, na pewno przeładowałeś zasób po wprowadzeniu poprawek?

**Comment by VVrongVVay** [comment]
Tak, mam prośbę. Może Pan mi te linijki dopisać i dać kod na pastebin?

W dniu sobota, 5 listopada 2016 XJMLN <notifications@github.com> napisał(a):

**Comment by XJMLN** [comment]
lss-bankomaty: http://pastebin.com/WLxx5ZCS
lss-trade:http://pastebin.com/TCCbcb5w

**Comment by VVrongVVay** [comment]
Już sprawdzam. A właśnie. Jak we frakcjach są niebieskie markery to czemu
one nie działają? Na jakiej to zasadzie jest lub powinno być?

2016-11-05 1:38 GMT+01:00 XJMLN <notifications@github.com>:

**Comment by VVrongVVay** [comment]
Kurcze.. wciąż da się do banku wpłacić 0.5 Przeładowałem skrypt wszystko
dodałem na pewno

2016-11-05 1:41 GMT+01:00 Michał Ulejczyk <vvrongvvay@gmail.com>:

**Comment by VVrongVVay** [comment]
To samo z przedmiotami.. przy wymianie.

W dniu 5 listopada 2016 01:44 użytkownik Michał Ulejczyk <
vvrongvvay@gmail.com> napisał:

**Comment by XJMLN** [comment]
Musi działać jeżeli u mnie na serwerze prywatnym działa, jeżeli posiadasz serwer publiczny - podaj IP wejdę i przetestuję.
(http://i.imgur.com/vwUe4EA.png)

**Comment by VVrongVVay** [comment]
mtasa://87.98.233.30:20144

W dniu 5 listopada 2016 01:46 użytkownik Michał Ulejczyk <
vvrongvvay@gmail.com> napisał:

**Comment by VVrongVVay** [comment]
Jednak dziś jestem w domu, więc na TS'ie również. GDayZ.gclan.pl

W dniu 5 listopada 2016 01:52 użytkownik Michał Ulejczyk <
vvrongvvay@gmail.com> napisał:

**Comment by VVrongVVay** [comment]
Mam również problem z odblokowaniem mapy..

W dniu 5 listopada 2016 09:40 użytkownik Michał Ulejczyk <
vvrongvvay@gmail.com> napisał:

**Comment by VVrongVVay** [comment]
http://pastebin.com/xQL4nGiA jak dodać by po uderzeniu baseballem był
paraliż?

W dniu 5 listopada 2016 15:01 użytkownik Michał Ulejczyk <
vvrongvvay@gmail.com> napisał:
